src/nvim/eval.c

```diff
@@ -7472,7 +7472,9 @@ static void f_and(typval_T *argvars, typval_T *rettv)
 static void f_api_info(typval_T *argvars, typval_T *rettv)
 {
   Dictionary metadata = api_metadata();
-  object_to_vim(DICTIONARY_OBJ(metadata), rettv, NULL);
+  if (!object_to_vim(DICTIONARY_OBJ(metadata), rettv, NULL)) {
```

Comment:
I could also have assigned it to the global `ignored`, but I thought asserting adds a bit more information about invariants.